Packages/vcs/Lib/VTKPlots.py

```diff
@@ -52,6 +52,8 @@ def __init__(self,canvas,renWin=None, debug=False,bg=None):
     self.numberOfPlotCalls = 0
     self.renderWindowSize=None
     self.clickRenderer = None
+    # Turn off anti-aliasing by default
+    self.antiAliasing = 0
```

Comment:
Let's just call it `self.antialiasing`, all lowercase (to match the set/get calls)